alerts.go

```diff
@@ -43,26 +44,66 @@ type Alert struct {
 	ClosedAt  int64   `json:"closedAt,omitempty"`
 }
 
-// FindAlerts find alerts
-func (c *Client) FindAlerts() ([]*Alert, error) {
-	req, err := http.NewRequest("GET", c.urlFor("/api/v0/alerts").String(), nil)
+// NextID is next id of alert
+type NextID string
+
+// Data includes alert and next id
+type Data struct {
+	Alerts []*Alert `json:"alerts"`
+	ID     NextID   `json:"nextId,omitempty"`
+}
+
+var d Data
```

Comment:
This makes FindAlerts~ goroutine unsafe.  Maybe declaring `d` inside `findAlertsWithParam` will solve.